Packs/Core/Integrations/CortexPlatformCore/CortexPlatformCore.py

```diff
@@ -493,13 +501,32 @@ def get_issue_recommendations_command(client: Client, args: dict) -> CommandResu
         "playbook_suggestions": playbook_suggestions,
     }
 
-    headers = [
-        "issue_id",
-        "issue_name",
-        "severity",
-        "description",
-        "remediation",
+    headers = ["issue_id", "issue_name", "severity", "description", "remediation"]
+
+    # Application Security issue
+    appsec_sources = [
+        "CAS_CVE_SCANNER",
+        "CAS_IAC_SCANNER",
+        "CAS_SECRET_SCANNER",
+        "CAS_LICENSE_SCANNER",
+        "CAS_SAST_SCANNER",
+        "CAS_OPERATIONAL_RISK_SCANNER",
+        "CAS_CI_CD_RISK_SCANNER",
+        "CAS_DRIFT_SCANNER",
     ]
+    if issue.get("alert_source") in appsec_sources:
```

Comment:
Move all this appSec code to a separate function.
Please check if someone already defined appsec_sources somewhere and better define this list as a in the top of of the file.